src/core/Akka/Event/LoggingBus.cs

```diff
@@ -88,18 +106,19 @@ protected override Type GetClassifier(object @event)
         internal void StartDefaultLoggers(ActorSystemImpl system)
         {
             var logName = SimpleName(this) + "(" + system.Name + ")";
-            var logLevel = Logging.LogLevelFor(system.Settings.LogLevel);
             var loggerTypes = system.Settings.Loggers;
             var timeout = system.Settings.LoggerStartTimeout;
-            var asyncStart = system.Settings.LoggerAsyncStart;
             var shouldRemoveStandardOutLogger = true;
 
+            LogLevel = Logging.LogLevelFor(system.Settings.LogLevel);
+
             foreach (var strLoggerType in loggerTypes)
             {
                 var loggerType = Type.GetType(strLoggerType);
                 if (loggerType == null)
                 {
-                    throw new ConfigurationException($@"Logger specified in config cannot be found: ""{strLoggerType}""");
+                    Publish(new Error(null, logName, GetType(), $@"Logger specified in config cannot be found: ""{strLoggerType}"""));
```

Comment:
We log an error instead of stopping the ActorSystem when a logger type can not be found

Reply:
If you consider a logger to be critical infrastructure, isn't this a bad idea? i.e. if I don't have visibility into how my service runs at a basic level, how can I be sure it's working correctly? 

I'd probably leave the old fail-fast mechanic in-place, which should be even rarer now with the changes in Akka.Hosting.

Reply:
This is different than the logger taking too long to startup - nothing is necessarily wrong with the users' configuration in that case. But in this case, the user's config is 100% wrong and they must fix it. That's why throwing an error here is a good idea.

Reply:
Alright, I'll bring the exception code back